lua/utils/init.lua

```diff
@@ -129,6 +131,23 @@ function utils.is_file(filename)
   return stat and stat.type == "file" or false
 end
 
+function utils.clone_table(t) -- deep-copy a table
```

Comment:
Doesn't `vim.deepcopy` answer your need ?

Reply:
It does! Thanks for the review as always.